docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -1,4 +1,144 @@
+  information are extended or withheld. Examples include varying accuracy in
+  face recognition, document search, or product recommendation.
+
+Fairness assessment and unfairness mitigation
+---------------------------------------------
+
+In Fairlearn, we provide tools to assess fairness of predictors for
+classification and regression. We also provide tools that mitigate unfairness
+in classification and regression. In both assessment and mitigation scenarios,
+fairness is quantified using disparity metrics as we describe below.
+
+Group fairness, sensitive features
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are many approaches to conceptualizing fairness. In Fairlearn, we follow
+the approach known as group fairness, which asks: *Which groups of individuals
+are at risk for experiencing harms?*
+
+The relevant groups (also called subpopulations) are defined using **sensitive
+features** (or sensitive attributes), which are passed to a Fairlearn
+estimator as a vector or a matrix called :code:`sensitive_features` (even if it is
+only one feature). The term suggests that the system designer should be
+sensitive to these features when assessing group fairness. Although these
+features may sometimes have privacy implications (e.g., gender or age) in
+other cases they may not (e.g., whether or not someone is a native speaker of
+a particular language). Moreover, the word sensitive does not imply that
+these features should not be used to make predictions – indeed, in some cases
+it may be better to include them.
+
+Fairness literature also uses the term *protected attribute* in a similar
+sense as sensitive feature. The term is based on anti-discrimination laws
+that define specific *protected classes*. Since we seek to apply group
+fairness in a wider range of settings, we avoid this term.
+
+Parity constraints
+^^^^^^^^^^^^^^^^^^
+
+Group fairness is typically formalized by a set of constraints on the behavior
+of the predictor called **parity constraints** (also called criteria). Parity
+constraints require that some aspect (or aspects) of the predictor behavior be
+comparable across the groups defined by sensitive features.
+
+Let :math:`X` denote a feature vector used for predictions, :math:`A` be a
+single sensitive feature (such as age or race), and :math:`Y` be the true
+label. Parity constraints are phrased in terms of expectations with respect to
+the distribution over :math:`(X,A,Y)`.
+For example, in Fairlearn, we consider the following types of parity constraints.
+
+*Binary classification*:
+
+* *Demographic parity* (also known as *statistical parity*): A classifier
+  :math:`h` satisfies demographic parity under a distribution over
+  :math:`(X, A, Y)` if its prediction :math:`h(X)` is statistically
+  independent of the sensitive feature :math:`A`. This is equivalent to
+  :math:`\E[h(X) \given A=a] = \E[h(X)] \quad \forall a`.
+  `[Agarwal, Beygelzimer, Dudik, Langford, Wallach] <https://arxiv.org/pdf/1803.02453.pdf>`__
```

Comment:
Can I suggest adding the year to all paper citations? Unless that's not the done thing in CS?

Reply:
There are as many citation styles as there are definitions of what constitutes a sandwich :-) I think it mainly depends on what people like. I've certainly seen all kinds of variations. 
[I personally always prefer seeing all authors since it gives me a better intuition on what paper it was. We had two different versions of Agarwal et al. in the terminology page, for example, which isn't terribly helpful until you click on the link.]
What we should be doing is define a standard format that every citation should abide by. Let's just do whatever scikit-learn does.

Reply:
Personally, I always let BibTeX do its thing :-)

I'd be in favour of including the year, since sometimes the same set of collaborators do a series of papers.

Actually, do we want a bibliography page instead?

Reply:
BibTex also just takes a configuration, so it depends on what document type you use... you probably just use the default of your document type without realizing it.

Scikit-learn does the following:
- inline it's just `Wu et al. (2004).`
- and it's linked in the "references" section per subsection https://scikit-learn.org/stable/modules/svm.html#scores-and-probabilities 
![image](https://user-images.githubusercontent.com/10245648/81028002-9b1c7b00-8e34-11ea-955c-1e8ecc74b130.png)

Reply:
As you can see in https://raw.githubusercontent.com/scikit-learn/scikit-learn/master/doc/modules/svm.rst

we use two styles of referencing:

```
.. _`liblinear`: https://www.csie.ntu.edu.tw/~cjlin/liblinear/

.. topic:: References:

   .. [#1] Platt `"Probabilistic outputs for SVMs and comparisons to
      regularized likelihood methods"
      <https://www.cs.colorado.edu/~mozer/Teaching/syllabi/6622/papers/Platt1999.pdf>`_.
```
and in the `.rst` code, we either have `[#1]_` or:

    `liblinear`_

rendered version: https://scikit-learn.org/dev/modules/svm.html

Reply:
I've copied that pattern and got a very surprising result! Still trying to determine why/how
Output inline:
![image](https://user-images.githubusercontent.com/10245648/81108706-e3d44280-8ecd-11ea-9438-ce1425f2553f.png)

references section: [note that 4 and 6 were cited twice, so this seems to be an odd way of handling the formatting... I don't like it.]
![image](https://user-images.githubusercontent.com/10245648/81108747-efc00480-8ecd-11ea-92ba-02a8ffd1d4f8.png)

Reply:
Wandering off-topic, I do remember a few `\bibliographystyle{}` commands too. As you say, most of the time, I was using that set by the `\documentclass{}` though :-)

Actually, just searching a little, perhaps:
https://sphinxcontrib-bibtex.readthedocs.io/en/latest/quickstart.html
would be helpful?

Reply:
Looks like this is the intended pattern. We can always adapt it in the future, of course.